internal/cmd/controller/gitops/reconciler/gitjob_controller.go

```diff
@@ -141,8 +144,9 @@ func (r *GitJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 	// Restrictions / Overrides, gitrepo reconciler is responsible for setting error in status
 	oldStatus := gitrepo.Status.DeepCopy()
-	gitrepo, err := grutil.AuthorizeAndAssignDefaults(ctx, r.Client, gitrepo)
+	_, err := grutil.AuthorizeAndAssignDefaults(ctx, r.Client, gitrepo)
 	if err != nil {
+		r.Recorder.Event(gitrepo, fleetevent.Warning, "Failed", err.Error())
```

Comment:
The `EventRecorder` [documentation](https://github.com/kubernetes/client-go/blob/master/tools/record/event.go#L98) states that the `reason` behind an event should be short and _unique_ (my emphasis).
Does this mean that each of our calls to `r.Recorder.Event` should provide a different reason?
eg. this one could be `FailedToApplyRestrictions`, whereas the one on L184 could be something like `FailedToCheckPolling`... of course, depending on how short we want to make them, there may be better examples.

Reply:
Yeah, good point, I also thought about it. 
I don't know if we should go further into detail we give to the user. 
I can try as you suggest...

Reply:
Is the remaining `Failed` a leftover here?

Reply:
yup, it was a leftover. Changed now